src/common/config/KBoxConfig.h

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
I have a separate PR coming up for configuration. I worked on it yesterday but did not have time to finish today. Maybe we can remove config stuff from this PR and just focus on the IMU with one fixed position?

Reply:
great!!
Since I got some home works to do at my PR anyway :-)
But yes, we can remove all the config stuff, no problem.